server/middlewares.js

```diff
@@ -137,6 +137,38 @@ const addTitle = (title) => {
     next();
   }
 }
+
+const handleUncaughtError = (error, req, res, next) => {
+    if (error.name === 'FetchError')
+    {
```

Comment:
We usually put braces on the previous line. http://eslint.org/docs/rules/brace-style#tbs